### ISSUE TITLE
"cannot connect to X server" on ubuntu 16.04

### ISSUE THREAD

**Issue by SanjayGhanagiri** [lllyasviel/PaintingLight#21]
**Issue :**

	"cannot connect to X server" on ubuntu 16.04

**Summary:**

	Running default.py with custom image or directly trying the examples results in the above error.
	Ubuntu desktop with 16 04 version was setup as per : https://gist.github.com/pseudokool/c7707937f075dd74cfa905edc80de39b

	This issue appears to be happening on lines 209-210 in ProjectPaintingLight.py where mouse is being accessed.

**Installed versions:**

	Ubuntu : 16.04
	Python : 3.6

	As per here : https://github.com/lllyasviel/PaintingLight/issues/13

	I have installed below specific version of required libraries in the same sequence.

	pip install opencv-python==4.2.0.34
	pip install opencv-contrib-python==4.2.0.34
	pip install h5py==2.10.0

	pip install tensorflow==1.4.0
	pip install scipy==1.1.0
	pip install trimesh==2.37.1

**Steps to replicate:**

	python default.py test-img.png
	python example001.py

**Error logs**
```

	2023-02-17 05:45:06.736604: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
	WARNING:tensorflow:No training configuration found in save file: the model was *not* compiled. Compile it manually.
	model-output<tensorflow.python.keras._impl.keras.engine.training.Model object at 0x7f2d8d56b2e8>
	Begin ray intersecting ...
	Intersecting finished.
	: cannot connect to X server :0
```
@lllyasviel @lllyasviel @78Alpha @jeexop @lrisviel @jeexop

**Comment by 78Alpha** [comment]
Looking over that gist, are you running a remote instance?

And for a sanity check, can it run any other GUI applications?

**Comment by SanjayGhanagiri** [comment]
@78Alpha yes am using a remote instance. Am able to run other GUI applications.
